app/views/shared/_navbar.html.erb

```diff
@@ -10,25 +10,17 @@
             <%= current_user.email %> <span class="caret"></span>
           </button>
           <ul class="dropdown-menu dropdown-menu-right">
-            <li>
-              <%= link_to 'Your Profile', user_path(current_user.id) %>
-            </li>
-            <li>
-              <%= link_to 'Dashboard', user_path(current_user.id) %>
-            </li>
+            <% if controller_name == "learn" %>
+              <li><%= link_to 'Pause Session', "/users/#{@user.id}/#{@deck.id}/preview" %></li>
+            <% end %>
```

Comment:
agree with all of the above - will remove for now and make a ticket for further consideration